docs/_dg/UserStories.md

```diff
@@ -1,47 +1,31 @@
 <!-- markdownlint-disable-file first-line-h1 -->
 
-**Add Inventory Item:**
+Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
```

Comment:
There seems to be an inconsistency here, do we use `* * *` or `***` for high priority?

Reply:
Ok will standardize to `* * *`